apps/bfd-shared-test-utils/src/main/java/gov/cms/bfd/DatabaseTestUtils.java

```diff
@@ -105,18 +116,27 @@ public static DataSource initUnpooledDataSource() {
     Optional<Properties> bfdServerTestDatabaseProperties = readTestDatabaseProperties();
 
     String url, username, password;
-    if (bfdServerTestDatabaseProperties.isPresent()) {
+    url = System.getProperty("its.db.url", "");
+    String usernameDefault = null;
+    String passwordDefault = null;
+
+    if (url != "" && url.endsWith("tc")) {
+      // Build the actual DB connection properties to use.
+      username = System.getProperty("its.db.username", usernameDefault);
+      if (username != null && username.trim().isEmpty()) username = "bfd";
+      password = System.getProperty("its.db.password", passwordDefault);
+      if (password != null && password.trim().isEmpty()) password = "bfdtest";
```

Comment:
Curious about maybe creating another variable instead of hardcoding username and password.

Reply:
done